src/app/components/ConsentBanner/index.jsx

```diff
@@ -0,0 +1,151 @@
+import React from 'react';
+import { string, element } from 'prop-types';
+import styled from 'styled-components';
+import {
+  C_CONSENT_BACKGROUND,
+  C_CONSENT_ACTION,
+  C_CONSENT_CONTENT,
+  C_WHITE,
+} from '@bbc/psammead-styles/colours';
+import {
+  GEL_GREAT_PRIMER,
+  GEL_LONG_PRIMER,
+  GEL_FF_REITH_SANS,
+} from '@bbc/gel-foundations/typography';
+import {
+  GEL_GROUP_3_SCREEN_WIDTH_MIN,
+  GEL_GROUP_2_SCREEN_WIDTH_MIN,
+} from '@bbc/gel-foundations/breakpoints';
+import {
+  GEL_MARGIN_BELOW_400PX,
+  GEL_MARGIN_ABOVE_400PX,
+} from '@bbc/gel-foundations/spacings';
+
+const Wrapper = styled.div`
+  font-family: ${GEL_FF_REITH_SANS};
+  background-color: ${C_CONSENT_BACKGROUND};
+  padding: 16px ${GEL_MARGIN_BELOW_400PX};
```

Comment:
```suggestion
  padding: 1rem ${GEL_MARGIN_BELOW_400PX};
```

Reply:
```suggestion
  padding: ${GEL_SPACING_DBL} ${GEL_MARGIN_BELOW_400PX};
```

Better yet use the const.